### ISSUE TITLE
cannot install dev on windows

### ISSUE THREAD

**Issue by CorradoLanera** [IndrajeetPatil/ggstatsplot#409]
## System info

OS: win 10 pro
R: 3.6.3 (tested on both 32 and 64 bit)
RStudio: 1.3.926
rtools: 3.5 (full)

## procedure(s)

both

```r
remotes::install_github(
  repo = "IndrajeetPatil/ggstatsplot", # package path on GitHub
  dependencies = FALSE, # assumes you have already installed needed packages
  quick = TRUE # skips docs, demos, and vignettes
)
```
```r
remotes::install_github(
  repo = "IndrajeetPatil/ggstatsplot", # package path on GitHub
  dependencies = TRUE, # installs packages which ggstatsplot depends on
  upgrade_dependencies = TRUE # updates any out of date dependencies
)
```

## output

```
> remotes::install_github(
+   repo = "IndrajeetPatil/ggstatsplot", # package path on GitHub
+   dependencies = FALSE, # assumes you have already installed needed packages
+   quick = TRUE # skips docs, demos, and vignettes
+ )
Using github PAT from envvar GITHUB_PAT
Downloading GitHub repo IndrajeetPatil/ggstatsplot@master
√  checking for file 'C:\Users\corra\AppData\Local\Temp\RtmpQdOSrj\remotes439475b61fe8\IndrajeetPatil-ggstatsplot-6bb51c9/DESCRIPTION' (432ms)
-  preparing 'ggstatsplot': (5.9s)
√  checking DESCRIPTION meta-information ...
-  checking for LF line-endings in source and make files and shell scripts (954ms)
-  checking for empty or unneeded directories
-  looking to see if a 'data/datalist' file should be added
-  building 'ggstatsplot_0.4.0.9000.tar.gz'
   
Installing package into ‘C:/Users/corra/<prj_path>/renv/library/R-3.6/x86_64-w64-mingw32’
(as ‘lib’ is unspecified)
* installing *source* package 'ggstatsplot' ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
  converting help for package 'ggstatsplot'
    finding HTML links ... done
    Titanic_full                            html  
    VR_dilemma                              html  
    aesthetic_addon                         html  
    bugs_long                               html  
    bugs_wide                               html  
    cat_counter                             html  
    cat_label_df                            html  
    combine_plots                           html  
    combine_plots2                          html  
    ggbarstats                              html  
    ggbetweenstats                          html  
    ggcoefstats                             html  
    ggcoefstats_label_maker                 html  
    ggcorrmat                               html  
    ggdotplotstats                          html  
    gghistostats                            html  
    ggpiestats                              html  
    ggscatterstats                          html  
    ggsignif_adder                          html  
    ggstatsplot-package                     html  
    ggwithinstats                           html  
    grouped_ggbarstats                      html  
    grouped_ggbetweenstats                  html  
    grouped_ggcorrmat                       html  
    grouped_ggdotplotstats                  html  
    grouped_gghistostats                    html  
    grouped_ggpiestats                      html  
    grouped_ggscatterstats                  html  
    grouped_ggwithinstats                   html  
    grouped_list                            html  
    histo_labeller                          html  
    intent_morality                         html  
    iris_long                               html  
    line_labeller                           html  
    mean_ggrepel                            html  
    mean_labeller                           html  
    movies_long                             html  
    movies_wide                             html  
    reexports                               html  
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:33: file link '%$%' in package 'ipmisc' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:33: file link '%<-%' in package 'ipmisc' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:33: file link '%<>%' in package 'ipmisc' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:33: file link '%>%' in package 'ipmisc' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:33: file link 'as_tibble' in package 'ipmisc' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:33: file link 'tibble' in package 'ipmisc' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:35: file link 'pairwise_p' in package 'pairwiseComparisons' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:37: file link '%|%' in package 'rlang' does not exist and so has been treated as a topic
Rd warning: C:/Users/corra/AppData/Local/Temp/Rtmp6Ps7nQ/R.INSTALL10e8348b357a/ggstatsplot/man/reexports.Rd:37: file link '%||%' in package 'rlang' does not exist and so has been treated as a topic
    theme_ggstatsplot                       html  
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
*** arch - i386
Error: package or namespace load failed for 'ggstatsplot' in library.dynam(lib, package, package.lib):
 DLL 'glue' not found: maybe not installed for this architecture?
Error: loading failed
Execution halted
*** arch - x64
ERROR: loading failed for 'i386'
* removing 'C:/Users/corra/<prj_path>/renv/library/R-3.6/x86_64-w64-mingw32/ggstatsplot'
* restoring previous 'C:/Users/corra/<prj_path>/renv/library/R-3.6/x86_64-w64-mingw32/ggstatsplot'
Error: Failed to install 'ggstatsplot' from GitHub:
  (converted from warning) installation of package ‘C:/Users/corra/AppData/Local/Temp/RtmpQdOSrj/file439411b42c1a/ggstatsplot_0.4.0.9000.tar.gz’ had non-zero exit status
```

**Comment by IndrajeetPatil** [comment]
Hmm, the source of this seems to be that the `glue` package is not installed:

```r
Error: package or namespace load failed for 'ggstatsplot' in library.dynam(lib, package, package.lib):
 DLL 'glue' not found: maybe not installed for this architecture?
Error: loading failed
Execution halted
```

If you refresh your R session and do `install.packages("glue")`, what do you see?

If that doesn't work, again refresh the R session, and try 

```r
install.packages("glue", INSTALL_opts = c("--no-multiarch"))
```

**Comment by CorradoLanera** [comment]
Just for the track: 
I used both the prerelease of RStudio, and a project with `{renv}` activated.

I tried to re-install `{glue}` (that was already installed) and it does not work (throwing an error regarding the impossibility to update `{Rcpp}`. Hence, I started the standard R GUI, and call `update.packages(checkBuilt = TRUE)`, and retried after that. This time the problem was in updating `{htmltools}` from GitHub. I deleted it and manually `remotes::install_github("rstudio/htmltools")` (from R GUI). Next, I executed the complete installation procedure for `{ggstatsplot}` from R GUI and it finally succeeded.
